### ISSUE TITLE
Change in behavior of robot query command

### ISSUE THREAD

**Issue by cmungall** [ontodev/robot#232]
It looks like behavior of robot query changes recently

Previously the `--format` option was optional. The default format wrote URIs directly without the enclosing `<...>`

Now it's necessary to pass a the format option or else we get a 

```
java.lang.NullPointerException
        at java.util.Objects.requireNonNull(Objects.java:203)
        at org.apache.jena.riot.resultset.ResultSetWriterRegistry.lookup(ResultSetWriterRegistry.java:51)
        at org.apache.jena.riot.ResultSetMgr.write(ResultSetMgr.java:160)
        at org.obolibrary.robot.QueryOperation.writeResult(QueryOperation.java:300)
        at org.obolibrary.robot.QueryOperation.writeResult(QueryOperation.java:311)
        at org.obolibrary.robot.QueryOperation.maybeWriteResult(QueryOperation.java:257)
        at org.obolibrary.robot.QueryOperation.runSparqlQuery(QueryOperation.java:193)
        at org.obolibrary.robot.QueryOperation.runSparqlQuery(QueryOperation.java:215)
        at org.obolibrary.robot.QueryCommand.execute(QueryCommand.java:173)
        at org.obolibrary.robot.CommandManager.executeCommand(CommandManager.java:219)
        at org.obolibrary.robot.CommandManager.execute(CommandManager.java:161)
        at org.obolibrary.robot.CommandManager.main(CommandManager.java:121)
        at org.obolibrary.robot.CommandLineInterface.main(CommandLineInterface.java:53)
```

(this is on master)

This is resolved by adding `-f tsv`. However, this now writes URIs enclosed in `<...>`. This messes things up when we use robot prior to an `extract` command, as extract doesn't recognize the `<...>`, only bare URIs. This is of course easily fixed by post-processing the output but we need to have predictable output of commands to stop pipelines breaking.

This may well be from one of my own changes...

**Comment by jamesaoverton** [comment]
I must have introduced this in d17c1556cac6eacbb6336b56c23ae66f9deaf41b.

`--format` is still supposed to be optional but the defaulting logic must be wrong somewhere.

I don't know why TSV output would change, though. We're still using `ResultSetLang.SPARQLResultSetTSV` as the format.

**Comment by jamesaoverton** [comment]
I pushed ad3a9dea2679032fc638b1392cfebfaba1c0cd60 to fix the `--format` problem. For SELECT it defaults to CSV as before, which does not include angle brackets for URIs. Please confirm.

I also built ROBOT aee76da1e3880ef8742cd07ab8d07f80b13adcaa and the TSV output looks the same as current master. Now I think I misunderstood you.

**Comment by cmungall** [comment]
OK, this all makes sense. All this time I was using the default csv export, I was selecting for a single column, so I didn't think I cared about csv vs tsv. But it turns out these are also different w.r.t. URI quoting. Restoring csv as default fixes things for us.

I think having the URI quoting appear differently is a bit odd, but we have a separate ticket for URI and literal rendering in tsv/csv here: #176

**Comment by jamesaoverton** [comment]
Yes, we still need to fix #176. Closing this.
